boards/pinetime/Makefile.features

```diff
@@ -0,0 +1,8 @@
+CPU_MODEL = nrf52832xxaa
+
+# Put defined MCU peripherals here (in alphabetical order)
+#FEATURES_PROVIDED += periph_i2c
```

Comment:
This doesn't have to be commented out anymore now that the I2C config is included.

Reply:
yup